pkg/trace/stats/concentrator.go

```diff
@@ -6,9 +6,11 @@
 package stats
 
 import (
+	"github.com/DataDog/datadog-agent/pkg/trace/config"
```

Comment:
Please make sure to move these imports down with the right group in all files (I see it multiple places)